### PR TITLE
Attachment improvement - add filename+filesize

### PR DESCRIPTION
**Pull request by Cu8eR**
Add filename and filesize into thumbnail title for images. 

attachment

It could be a part of #1583 . If you dont like this PR, just reject this. I applied it into my board, so I made a PR.

### CODE REVIEW

**Comment by Sama34** [comment]
Seems like a nice addition.

File: install/resources/mybb_theme.xml [hunk 1]
```diff
@@ -9485,11 +9485,11 @@ if(use_xmlhttprequest == "1")
...
 		<template name="postbit_attachments_thumbnails" version="120"><![CDATA[<span class="smalltext"><strong>{$lang->postbit_attachments_thumbnails}</strong></span><br />
 {$post['thumblist']}
 <br />]]></template>
-		<template name="postbit_attachments_thumbnails_thumbnail" version="1800"><![CDATA[<a href="attachment.php?aid={$attachment['aid']}" target="_blank"><img src="attachment.php?thumbnail={$attachment['aid']}" class="attachment" alt="" title="{$attachdate}" /></a>&nbsp;&nbsp;&nbsp;]]></template>
+		<template name="postbit_attachments_thumbnails_thumbnail" version="1805"><![CDATA[<a href="attachment.php?aid={$attachment['aid']}" target="_blank"><img src="attachment.php?thumbnail={$attachment['aid']}" class="attachment" alt="" title="{$lang->postbit_attachment_filename} {$attachment['filename']}&#13{$lang->postbit_attachment_size} {$attachment['filesize']} / {$attachdate}" /></a>&nbsp;&nbsp;&nbsp;]]></template>
```
Comment (Destroy666x) [inline install/resources/mybb_theme.xml]: `$attachdate` could also be moved to a new line, `/` usually means `or` which is not the case here: http://en.wikipedia.org/wiki/Slash_%28punctuation%29#In_English_text

Comment (Cu8eR) [inline install/resources/mybb_theme.xml]: So you wanna this:

Filename: xxx
Size: xx kB
Date: (only date, right?)

Comment (Destroy666x) [inline install/resources/mybb_theme.xml]: Yes, something like that

Comment (Cu8eR) [inline install/resources/mybb_theme.xml]: Ok, I will fix my RP in 2 days

**Comment by Cu8eR** [comment]
@Destroy666x - FIXED :)

**Comment by Destroy666x** [comment]
The 2nd newline is not wroking because you're using improper `&#13` everywhere, they should be `&#13;`

**Comment by Cu8eR** [comment]
I am sorry - I edited it in GitHub online editor - I missed this

**Comment by Destroy666x** [comment]
https://github.com/mybb/mybb/issues/1877

**Comment by Sama34** [comment]
I created an issue for this. Please remember issues need to be created to keep a record.

**Comment by Sama34** [comment]
I will take my words back now.

**Pull request merged**